src/pad/src/ICeWall.cpp

```diff
@@ -1123,7 +1123,7 @@ std::set<odb::dbNet*> ICeWall::connectByAbutment(
 
 std::vector<std::pair<odb::dbITerm*, odb::dbITerm*>> ICeWall::getTouchingIterms(
     odb::dbInst* inst0,
-    odb::dbInst* inst1) const
+    odb::dbInst* inst1)
```

Comment:
Why did this stop being const?

Reply:
I needed it in psm, so I made it static instead